sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleCounterSdk.java

```diff
@@ -38,14 +38,14 @@ private DoubleCounterSdk(
 
   @Override
   public void add(double delta, LabelSet labelSet) {
-    BoundDoubleCounter boundDoubleCounter = bind(labelSet);
-    boundDoubleCounter.add(delta);
-    boundDoubleCounter.unbind();
+    BoundInstrument boundInstrument = bind(labelSet);
+    boundInstrument.add(delta);
+    boundInstrument.unbind();
   }
 
   @Override
-  public BoundDoubleCounter bind(LabelSet labelSet) {
-    return new BoundInstrument(labelSet, monotonic);
+  public BoundInstrument bind(LabelSet labelSet) {
+    return new BoundInstrument(monotonic);
```

Comment:
why did you stop passing in the labelset?

Reply:
The mapping between LabelSet and bound will be in the AbstractInstrumentWithBinding (from #831 ) so unnecessary memory usage in the Bound to also keep a reference to the LabelSet.